dev/misc_utils/git_jira_fix_version_check.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python
+############################################################################
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+############################################################################
+
+
+import inspect
+import os
+import re
+import subprocess
+
+from jira import JIRA
+
+first_exclude_commit_hash = input("First commit hash to start excluding commits from history: ")
+fix_version = input("Fix Version: ")
+
+jira = JIRA({"server": "https://issues.apache.org/jira"})
```

Comment:
As we'll soon need to release -queryserver and -connectors a well, can you make 
the repository directory and the JIRA project name into parameters ?

That way we can use this for other branches and projects without having to deuplicate it into each one.

Reply:
Makes sense, Tephra, Omid too can be released. Nice one, let me make Jira project name as well as absolute path of local project dir as parameters so that various branches and projects can be covered.

Reply:
Btw I believe this line might remain same because Jira server address remains same regardless of projects. However, `project name` and `repository directory` are parameters.

Reply:
Great thought! Let me keep it in parameter with default as ASF Jira url 👍